package.json

```diff
@@ -39,7 +39,7 @@
     "react-simple-dropdown": "3.2.3",
     "react-window": "1.3.1",
     "redoc": "2.0.0-rc.31",
-    "redoc-cli": "0.9.8",
+    "redoc-cli": "0.9.11",
```

Comment:
Is this the change that is causing GA to fail?

Reply:
I think it's a co-depdency of redoc itself.  I had it at redoc rc.37 which broke things.  Downgrading didn't help.  We can try and pull this version down too.  But why isn't Travis-CI complaining?